### PR TITLE
Add comma in `README` for clearer message

### PR DESCRIPTION
**Pull request by Rudxain**
I think it makes it less ambiguous

### CODE REVIEW

**Review by r10s — COMMENTED** [review]
lgtm - but that does not mean much in this case, my english is not good enough to decide on that comma :) cc @rosano .

just a general note: meanwhile also Sequoia has a more libre license, LGPL , https://sequoia-pgp.org/blog/2021/10/18/202110-sequoia-pgp-is-now-lgpl-2.0/ - ianal, however, i think MIT/Apache is still more permissive so that sentence is still correct.

**Comment by Rudxain** [comment]
Interesting, thank you for the info

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ And then run tests with `RUST_LOG=pgp=info`.
 
 Some key differences:
 
-- rPGP has a more libre license than Sequoia that allows a broader usage
+- rPGP has a more libre license than Sequoia, that allows a broader usage
```
Comment (rosano) [inline README.md:62]: I think this is more gramatic

```suggestion
- rPGP has a more libre license than Sequoia, which allows a broader usage
```

**Review by r10s — APPROVED** [review]

**Comment by dignifiedquire** [comment]
thanks

**Pull request merged**